### ISSUE TITLE
Provide a way to list existing timers

### ISSUE THREAD

**Issue by jonhoo** [andrewjstone/ferris#4]
The API currently doesn't allow clients to inspect the list of timers in any way, even for debugging. Implementing `fmt::Debug` and/or providing a `.timers()` method that gives an iterator over the registered timers would be extremely handy.

**Comment by andrewjstone** [comment]
I agree this would be handy for debugging. I'll make sure it gets implemented at some point.
